packages/react-canvas-core/src/core-actions/ActionEventBus.ts

```diff
@@ -80,4 +80,10 @@ export class ActionEventBus {
 			action.options.fire(actionEvent as any);
 		}
 	}
+
+	clearKeys() {
+		_.forEach(Object.keys(this.keys), (key) => {
```

Comment:
dont need the Object.keys, `_.forEach` will give the key as the second param in the callback